common/utils/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -100,7 +99,7 @@ case class MessageWithContext(message: String, context: java.util.HashMap[String
  * Companion class for lazy evaluation of the MessageWithContext instance.
  */
 class LogEntry(messageWithContext: => MessageWithContext) {
-  def message: String = StringEscapeUtils.unescapeJava(messageWithContext.message)
```

Comment:
Should we maybe simply try-catch, and return the original string if it fails?

Reply:
Windows users are arguably less in any event.

Reply:
and logging shouldn't fail the application anyway.

Reply:
Hmm..., this may have `successfully unescaped`, but the result `does not` meet expectations, such as:
```
println(StringEscapeUtils.unescapeJava("C:\\Users\\runneradmin\\AppData\\Local\\remp\\spark-9f4b7e74-ea9e-46fc-bde6-1449cc9f6e8e"))
```

We may expect it to output as:
```
C:\Users\runneradmin\AppData\Local\remp\spark-9f4b7e74-ea9e-46fc-bde6-1449cc9f6e8e
```

Actual, its output is:
```
empspark-9f4b7e74-ea9e-46fc-bde6-1449cc9f6e8e
```

So it's better not to execute the `unescape` operation and let the user manually use the method I wrote in `UT` to achieve its expected behavior, although it does look a bit ugly.

Reply:
> and logging shouldn't fail the application anyway.

Yes, we can use `try ... catch` temporarily fixes it, but it has some of my flaws mentioned above.
Or should we do this (use `try ... catch`) first and let `build_sparkr_window` pass?

Reply:
Hm.

Reply:
I am fine with this approach but I think we need @gengliangwang 's look at least.

Reply:
Yeah, let's wait for @gengliangwang's review. 
Thanks!